go-controller/pkg/config/config.go

```diff
@@ -719,3 +720,53 @@ func (a *OvnDBAuth) SetDBAuth() error {
 	}
 	return nil
 }
+
+// UpdateOvnNodeAuth updates the host and URL in ClientAuth and ServerAuth
+// for both OvnNorth and OvnSouth. It updates them with the new masterIP.
+func UpdateOvnNodeAuth(masterIP string) error {
```

Comment:
@dcbw 
Would you mind looking at this function. We have some indirections added for this and whether doing something like this is okay with your idea of why those indirections were introduced.